test/integration/routes.test.js

```diff
@@ -13,6 +18,7 @@ const { AUTH_ISSUER, AUTH_AUDIENCE } = require('../../app/env')
 const { cleanup } = require('../seeds/members')
 
 describe('routes', function () {
+  this.timeout(3000)
```

Comment:
If anyone asks needed to increase as default is 1s and my PC is to slow :sad_parrot:

Reply:
this is ok. Yes sad but it's integration tests in blockchain so not overly surprising